frontends/web/src/components/status/status.tsx

```diff
@@ -45,7 +45,7 @@ export const Status = ({
   const checkConfig = useCallback(async () => {
     if (dismissible) {
       const config = await getConfig();
-      setShow(!config ? true : !config.frontend[dismissible]);
+      setShow(!config?.frontend ? true : !config.frontend[dismissible]);
```

Comment:
In many other places we check only for config and if defined assume that there is `config.frontend`. How  can this happen / could you add steps to reproduce?

alternatively, couldn't the backend ensure that there is always a frontend property with at least an empty object?

Reply:
I noticed an error caused by this a few days ago, but could not reproduce it anymore. I opened the PR anyway as it was quick, but yeah if there are other places with the same check it could make sense to address it backend side

Reply:
Didn't have time yet, but I'll look into the backed part to see if we can fix it there